packages/client/src/components/app/Accordion.svelte

```diff
@@ -0,0 +1,11 @@
+<script>
+  import { Accordion } from "@budibase/bbui"
+
+  export let bold
+  export let initialOpen
+  export let label
+</script>
+
+<Accordion header={label} {bold} {initialOpen}>
```

Comment:
We should change this to `header={label || ""}` just to ensure it doesn't say `undefined` when there's no text.